### PR TITLE
feat(python): next_event support both timedelta and int

### PR DESCRIPTION
**Pull request by itayd**
i think will be less confusing.

### CODE REVIEW

**Review by efiShtain — APPROVED** [review]
fix the else / raise issue

**Comment by itayd** [comment]
> fix the else / raise issue

done

File: runtimes/pythonrt/py-sdk/autokitteh/events.py [hunk 1]
```diff
@@ -18,7 +18,9 @@ def unsubscribe(subscription_id: str) -> None:
     pass
 
 
-def next_event(subscription_id: str | list, *, timeout: timedelta = None) -> AttrDict:
+def next_event(
+    subscription_id: str | list[str], *, timeout: timedelta | int = None
```
Comment (tebeka) [inline runtimes/pythonrt/py-sdk/autokitteh/events.py:22]: Why int and not float? I can't do a 500ms timeout.

Comment (itayd) [reply runtimes/pythonrt/py-sdk/autokitteh/events.py:22]: right! will fix

Comment (itayd) [reply runtimes/pythonrt/py-sdk/autokitteh/events.py:22]: see https://github.com/autokitteh/autokitteh/pull/815

**Pull request merged**